worker_test.go

```diff
@@ -322,18 +589,17 @@ func TestResourceConversion(tt *testing.T) {
 				scenario.typeLB,
 				clusterLocalDomain,
 				true)
-			if err != nil {
-				t.Fatal(err)
-			}
+			require.NoError(t, err)
+
 			log.SetLevel(log.DebugLevel)
 			problems := doWork(&mockedCertificateProvider{t: t}, 10, time.Hour, a, k, "")
-			if len(problems.Errors()) > 0 {
-				t.Error(problems.Errors())
-			}
+			require.Empty(t, problems)
+
+			// TODO: assert on number of stacks created (currently 1) to check non-shared load balancer
 
 			assert.Equal(
 				t,
-				expected,
+				strings.TrimSuffix(expected, "\n"),
```

Comment:
to make git and tests happy

Reply:
It is better to trim it where `expected` is defined